packages/core/src/lib/util.ts

```diff
@@ -140,7 +142,17 @@ export function findSuitableIcon(
 
   let largestIcon: WebManifestIcon | null = null;
   let largestIconSize = 0;
+  
   for (const icon of icons) {
+    // Use the mime-type from the icon or look up from the URL if one is not provided.	
+    const mimeType = icon.mimeType || lookup(icon.src);
+
+    // We don't support SVG images, so skip SVG icons.	
```

Comment:
This comment is a little misleading - maybe something like "ignore SVG icons if SVG support isn't set up".